app/components/osf-navbar/component.ts

```diff
@@ -59,22 +60,25 @@ export default class OsfNavbar extends Component.extend() {
         return appName.toUpperCase();
     });
 
-    actions = {
-        ...this.actions, // from AnalyticsMixin
+    @action
+    toggleSearch() {
+        this.toggleProperty('showSearch');
+        this.send('closeSecondaryNavigation');
+    }
 
-        toggleSearch() {
-            this.toggleProperty('showSearch');
-            this.send('closeSecondaryNavigation');
-        },
-        closeSecondaryNavigation() {
-            this.$('.navbar-collapse').collapse('hide');
-        },
-        closeSearch() {
-            this.set('showSearch', false);
-        },
-        closeSecondaryAndSearch() {
-            this.send('closeSecondaryNavigation');
-            this.send('closeSearch');
-        },
-    };
+    @action
+    closeSecondaryNavigation() {
+        $('.navbar-collapse').collapse('hide');
```

Comment:
Any chance your reload problem is related to switching from `this.$` to `$`? `this.$` might be better anyway, since it's scoped to this component rather than searching the whole page.

Reply:
No, the reload was happening specifically with this.$, it's just that this.$ definitely wasn't working on the home-links (because they are tagless components https://github.com/CenterForOpenScience/ember-osf-web/pull/154/files#diff-c52513383b0fd32f9db3e02113493d43R12 ). 

In fact the closing secondary nav wasn't working here until I changed it, possibly because it's, in this case, being used by something that's outside of the hierarchy of things that were being collapsed (the home link instead of one of the links inside the menu).